spec/models/page_spec.rb

```diff
@@ -359,4 +359,56 @@
       expect(expected_items.first.value).to eq('123')
     end
   end
+
+  describe '#contains_placeholders?' do
+    context 'when page has placeholders' do
+      subject(:page) { service.find_page_by_url('privacy') }
+
+      before do
+        allow(page).to receive(:placeholders).and_return(['[insert your organisation name]'])
```

Comment:
_super_ picky comment but it's probably worth having another item in this placeholder array to test your join in the regex generation